src/app/js/public/Toolbar.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import FacetList from './facet/FacetList';
 import Filter from './dataset/Filter';
 
-export const ToolbarComponent = () => (
+export const Toolbar = ({ name }) => (
     <div>
-        <Filter />
+        <Filter key={name} />
```

Comment:
Protip: By adding the graph page name as the key of this component, it is resetted each time the page change (eg. from `/graph` to `/graph/abcd`)

Reply:
:+1: